packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -788,13 +788,38 @@ class _CupertinoBackGestureController<T> {
     }
 
     if (controller.isAnimating) {
+      late VoidCallback animationValueCallback;
+      late AnimationStatusListener animationStatusCallback;
```

Comment:
nit: consider using an inline function for late binding, so the binding is also final.